README.md

```diff
@@ -130,43 +130,48 @@ With your own parameters:
 
 <h2 align="center">Main Features</h2>
 
-**Splitting video by scenes for parallel encoding** because AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
+Av1an allows for **splitting input video by scenes for parallel encoding** to improve encoding performance because most AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
 
 - [Vapoursynth](http://www.vapoursynth.com) script input support.
 - Speed up video encoding.
-- Target Quality mode. Targeting end result reference visual quality. VMAF used as a substructure
+- "Target Quality" mode. Targeting end result reference visual quality. VMAF used as a substructure
 - Resuming encoding without loss of encoded progress.
 - Simple and clean console look.
 - Automatic detection of the number of workers the host can handle.
 - Both video and audio transcoding.
 
-## Install
+## Installation
 
+Av1an can be installed in two main ways, either:
 - With a package manager:
   - Cargo: `cargo install av1an`
   - Arch Linux: `pacman -S av1an`
+    
+- Or do it manually:
+  - Clone repository or download from [Releases](https://github.com/master-of-zen/Av1an/releases), then do `cargo build --release`.
 
-- Prerequisites:
+- Make sure to install these prerequisites first:
   - [Install FFmpeg](https://ffmpeg.org/download.html)
-  - [Vapoursynth](http://www.vapoursynth.com/)
-  - Recommended to install lsmash/ffms2 for faster and better processing
+  - [Install Vapoursynth](http://www.vapoursynth.com/)
+  - Recommended to install lsmash/ffms2 for faster and better pre-processing.
 
-- Encoder of choice:
-  - [Install AOMENC](https://aomedia.googlesource.com/aom/)
-  - [Install rav1e](https://github.com/xiph/rav1e)
+An encoder is also required, install any of these depending on the format/codec:
+- For AV1:
+  - [Install aomenc](https://aomedia.googlesource.com/aom/)
   - [Install SVT-AV1](https://gitlab.com/AOMediaCodec/SVT-AV1)
-  - [Install vpx](https://chromium.googlesource.com/webm/libvpx/) VP9, VP8 encoding
-
-- Optional :
-  - [ffms2](https://github.com/FFMS/ffms2)
-  - [lsmash](https://github.com/VFR-maniac/L-SMASH-Works)
-  - [mkvmerge](https://mkvtoolnix.download/)
-
-- Manually:
-  - Clone Repo or Download from Releases
-  - `cargo build --release`
-
-## Docker
+  - [Install rav1e](https://github.com/xiph/rav1e)
+- For VP9/VP8:
+  - [Install libvpx](https://chromium.googlesource.com/webm/libvpx/)
+
+Av1an also supports these optional components:
+- Chunking components:
+  - [Install ffms2](https://github.com/FFMS/ffms2)
+  - [Install lsmash](https://github.com/VFR-maniac/L-SMASH-Works)
```

Comment:
This reminds me that it might be useful to clarify that on Arch linux, you have to install the `vapoursynth-plugin-lsmashsource` package to install lsmash as opposed to the `l-smash` package, since I have seen people confused about that before.

Reply:
I also would like to add that in Fedora, installing nasm, llvm, and clang is required. I've experienced that exact issue when building from source for the first time. Maybe the installation section should be further written for various Linux distributions?

Reply:
That sounds good, although for now I think it's also fine if we just mention that nasm, llvm and clang are required if you want to do that instead

Reply:
For now that would probably be a good idea, but beside that I really don't know if there are other dependencies that are required beside the ones that I've mentioned above,